src/components/FilmsList.vue

```diff
@@ -1,23 +1,26 @@
 <template>
   <div>
-    <div class="d-flex flex-column justify-content-center align-items-end container" v-if="!loading">
+    <div
+      class="d-flex flex-column justify-content-center align-items-end container mt-3"
+      v-if="!isLoading"
+    >
       <div class="d-flex flex-row">
         <b-list-group horizontal class="flex-sm-wrap">
           <b-list-group-item
-              class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
-              v-for="movie in uniqueMovies"
-              :key="movie.id"
+            class="w-auto m-2 p-0 border-0 d-flex justify-content-center align-items-start rounded-circle card"
+            v-for="movie in uniqueMovies"
```

Comment:
You can destructure the `movie` parameter as `{ title, backdrop_path, overview... }`, etc.